CHANGELOG.rst

```diff
@@ -2,6 +2,48 @@
 Change history
 ==============
 
+2.0.0 (2021-09-22)
```

Comment:
Regarding versioning: Let's just use (only) the highest API version as component version.

This means version 2.0.0 of the Objects API supports Objects API specification 1.2.0 and 2.0.0.
The changelog item can therefore be merged, with the mention of the support for both API spec versions.

Also, let's explicetly, mentioned API version changes below this version, so:

2.0.0 (2021-09-22)
------------------

**Component changes**

* Supports API 2.0.0 and API 1.2.0

**API 1.2.0 changes**

* **Breaking features**
  * paginated API responses (#148)
  * ...

**API 2.0.0 changes**

...